ItemList/ItemList/ViewModel.cs

```diff
@@ -0,0 +1,146 @@
+        {
+            return new RelayCommand(obj =>
+            {
+                this.ListOfItems.Add(new Phone { Manufacture = this.Manufacture, Model = this.Model });
+                ClearField();
+                UpdateTextBoxes();
+            });
+        }
+        /// <summary>
+        /// Method is used for clearing text boxes.
+        /// </summary>
+        /// <owner>Anton Petrenko</owner>
+        private void ClearField()
+        {
+            Manufacture = string.Empty;
+            Model = string.Empty;
+        }
+        /// <summary>
+        /// Calls OnPropertyChanged event
+        /// </summary>
+        /// <owner>Anton Petrenko</owner>
+        private void UpdateTextBoxes()
+        {
+            OnPropertyChanged("Manufacture");
+            OnPropertyChanged("Model");
+        }
+        /// <summary>
+        /// Gets delete command for collection of phones.
+        /// </summary>
+        /// <owner>Anton Petrenko</owner>
+        /// <returns>Instance of RelayCommand class.</returns>
+        public RelayCommand DeleteCommand
+        {
+            get
+            {
+                return this.deleteCommand ??
+                    (this.deleteCommand = new RelayCommand(obj =>
+                    {
+                        this.ListOfItems.Remove(this.selectedItem);
+                    }));
+            }
+        }
+        /// <summary>
+        /// Gets or sets collection of items.
+        /// </summary>
+        /// <owner>Anton Petrenko</owner>
+        public ObservableCollection<Phone> ListOfItems { get; set; }
+        /// <summary>
+        /// Gets or sets manufacture of the item.
+        /// </summary>
+        /// <owner>Anton Petrenko</owner>
+        /// <returns>Manufacture name as a string.</returns>
+        public string Manufacture { get; set; }
+        /// <summary>
+        /// Gets or sets model of the item.
+        /// </summary>
+        /// <owner>Anton Petrenko</owner>
+        /// <returns>Model name as a string.</returns>
+        public string Model { get; set; }
+        /// <summary>
+        /// Calls PropertyChanged event.
+        /// </summary>
+        /// <owner>Anton Petrenko</owner>
+        public void OnPropertyChanged([CallerMemberName]string prop = "")
+        {
+            if (PropertyChanged != null)
+                PropertyChanged(this, new PropertyChangedEventArgs(prop));
+        }
+        /// <summary>
+        /// Represents event type of PropertyChangedEventHandler.
+        /// </summary>
+        /// <owner>Anton Petrenko</owner>
+        public event PropertyChangedEventHandler PropertyChanged;
+        /// <summary>
+        /// Gets or sets selected item.
+        /// </summary>
+        /// <owner>Anton Petrenko</owner>
+        /// <returns>Instance of the phone.</returns>
+        public Phone SelectedItem
+        {
+            get { return selectedItem; }
+            set
+            {
+                this.selectedItem = value;
```

Comment:
lets have a look at the case.
Each time when the same value is specified, each time we will rise the PropertyChangedEvent.
Good approach is to check if the value has been changed:
if(this.selectedItem == value)
  return;
this.selectedItem = value
this.OnPropertyChanged(nameof(this.SelectedItem));

Reply:
how about that?            
set
            {
                if (this.selectedItem!=value)
                this.selectedItem = value;
                this.OnPropertyChanged("SelectedItem");
            }